bin/main.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+# Welcome to players
+    def welcome
+    puts "Welcome Players!"
+    puts "Tic Tac Toe is a game for 2 players."
+    puts "The game is played on a grid that's 3 x 3"
+    puts "If you are X, your friend? is O"
+    puts "Players take turns putting their marks in empty squares"
```

Comment:
this should be also mocked in here

* where is the game asking for a move to the players? 
* What are the possible moves available?
* if a player inputs a move that is not valid, how is the game going to react?